.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
```

Comment:
Since we build in Docker anyway, we could just use `ubuntu-latest` as well. One thing to worry about less.

Reply:
yea